app/assets/javascripts/city.js.coffee

```diff
@@ -0,0 +1,5 @@
+select_to_select2 = ->
+  $('#ad_city').select2( { tags: true } )
+
+$(document).on 'turbolinks:load', ->
+  select_to_select2()
```

Comment:
Can we remove this as we have already bind these ids into `dropdown.js.coffee` for select2.